src/ontology/cl-edit.owl

```diff
@@ -29464,7 +29464,7 @@ EquivalentClasses(obo:CL_4023039 ObjectIntersectionOf(obo:CL_0000540 ObjectSomeV
 # Class: obo:CL_4023040 (L2/3-6 intratelencephalic projecting glutamatergic neuron)
 
 AnnotationAssertion(obo:IAO_0000028 obo:CL_4023040 "L2/3-6 IT glut")
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1101/2020.10.19.343129") obo:IAO_0000115 obo:CL_4023040 "A intratelencephalic-projecting glutamatergic neuron with a soma found in cortical layers L2/3-6")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34616075") Annotation(oboInOwl:hasDbXref "https://cellxgene.cziscience.com/e/c3aa4f95-7a18-4a7d-8dd8-ca324d714363.cxg/") obo:IAO_0000115 obo:CL_4023040 "A intratelencephalic-projecting glutamatergic neuron with a soma found in cortical layers L2/3-6.")
```

Comment:
There is a grammatical mistake: `A intratelencephalic` &rarr; `An intratelencephalic`